src/web/app/components/logs-table/logs-table.component.html

```diff
@@ -21,7 +21,24 @@
           </tr>
           <tr *ngIf="log.isDetailsExpanded">
             <td colspan="3">
-              <pre *ngIf="log.isDetailsExpanded" [innerHTML]="log.details | json"></pre>
+              <div>
+                <details open>
+                  <summary class="title">source location:</summary>
+                  <pre class="clickable margin-left-3em wrap" (click)="addSourceLocationToFilter(log.sourceLocation)" [innerHTML]="log.sourceLocation | json"></pre>
+                </details>
+              </div>
+              <details open>
+                <summary class="title">details:</summary>
+                <pre class="margin-left-3em wrap" [innerHTML]="log.details | json"></pre>
+              </details>
+              <details open *ngIf="log.userInfo">
+                <summary class="title">userInfo:</summary>
+                <pre class="clickable margin-left-3em wrap" (click)="addUserInfoToFilter(log.userInfo)" [innerHTML]="log.userInfo | json"></pre>
+              </details>
+              <div>
+                <span class="title" *ngIf="log.traceId">trace: </span>
+                <span class="clickable" (click)="addTraceToFilter(log.traceId)">{{ log.traceId }}</span>
```

Comment:
Based on this and also the screenshot the filter is added upon clicking. Not right/wrong on its own, but it's not the most intuitive manner, especially since nothing in this displayed value indicates that there is an interaction when clicking (text doesn't look different, cursor doesn't change into pointer, etc; the highlight only appears on hover and it doesn't tell anything about being clickable).

Reply:
I have added a tooltip to let users know that the value can be clicked. The cursor will also change to pointer when hover over fields that can be clicked.
![image](https://user-images.githubusercontent.com/58677983/125740306-a32b0c73-c751-4200-b988-35147d9f9aa2.png)